### PR TITLE
Add opt-in feature to anonymize tweets

### PR DESCRIPTION
**Pull request by iGitScor**
**Feature proposal:** Anonymous tweets to encourage content

### CODE REVIEW

File: source/features/anonymize-tweets.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default function () {
+	$('.FullNameGroup .fullname').text('Jane Doe');
+	$('.js-retweet-text b').text('Jane Doe');
+	$('.UserBadges').hide();
+	$('.username').hide();
```
Comment (sindresorhus) [inline source/features/anonymize-tweets.js]: These two can be done call.
```suggestion
	$('. UserBadges, .username').hide();
```

File: source/features/anonymize-tweets.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export default function () {
+	$('.FullNameGroup .fullname').text('Jane Doe');
+	$('.js-retweet-text b').text('Jane Doe');
```
Comment (sindresorhus) [inline source/features/anonymize-tweets.js]: Same as https://github.com/sindresorhus/refined-twitter/pull/173/files#r267718771

Comment (sindresorhus) [reply source/features/anonymize-tweets.js]: How about `Math.random() >= 0.5 ? 'Jane Doe' : 'John Doe'`? That way we stay totally neutral.

File: source/features/index.js [hunk 3]
```diff
@@ -184,6 +184,13 @@ export const features = {
 		fn: require('./inline-instagram-photos').default,
 		hidden: true
 	},
+	janeDoeTweets: {
```
Comment (sindresorhus) [inline source/features/index.js]:
```suggestion
	anonymizeTweets: {
```

**Comment by sindresorhus** [comment]
Cool idea :)

Can you also add it to the readme?

File: source/features/anonymize-tweets.js (hunk 2 above)
Comment (iGitScor) [reply source/features/anonymize-tweets.js]: For sure, I changed it

File: source/features/index.js [hunk 4]
```diff
@@ -127,6 +127,13 @@ export const features = {
 	},
 
 	/* TIMELINE */
+	anonymizeTweets: {
+		id: 'feature-anonymous-tweets',
```
Comment (sindresorhus) [inline source/features/index.js]: This doesn't match the filename wording.

File: source/features/index.js [hunk 5]
```diff
@@ -127,6 +127,13 @@ export const features = {
...
+		category: 'timeline',
+		label: 'Transform your timeline in Jane Doe tweets timeline',
```
Comment (sindresorhus) [inline source/features/index.js]: Needs a better description. More like the one in the readme.

File: source/features/index.js [hunk 6]
```diff
@@ -127,6 +127,13 @@ export const features = {
 	},
 
 	/* TIMELINE */
+	anonymizeTweets: {
```
Comment (sindresorhus) [inline source/features/index.js]: This should be added to the bottom of the `TIMELINE` list of features.

File: readme.md [hunk 7]
```diff
@@ -42,6 +42,7 @@ We use Twitter a lot and notice many dumb annoyances we'd like to fix. So here b
 - Uses your personal color theme on all profiles.
 - Hides the header image on profile pages.
 - [Shows alternative image text below images when available.](https://user-images.githubusercontent.com/170270/40556400-b46c292c-6076-11e8-8241-f5c4e1a7a161.png)
+- Anonymizes tweets by replacing real usernames by "John Doe" or "Jane Doe"
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
- Anonymizes tweets by replacing real usernames with "John Doe" or "Jane Doe". *(Opt-in)*
```

**Review by jorgegonzalez — APPROVED** [review]

File: readme.md [hunk 8]
```diff
@@ -42,6 +42,7 @@ We use Twitter a lot and notice many dumb annoyances we'd like to fix. So here b
 - Uses your personal color theme on all profiles.
 - Hides the header image on profile pages.
 - [Shows alternative image text below images when available.](https://user-images.githubusercontent.com/170270/40556400-b46c292c-6076-11e8-8241-f5c4e1a7a161.png)
+- Anonymizes tweets by replacing real usernames with "John Doe" or "Jane Doe"
```
Comment (sindresorhus) [inline readme.md:45]:
```suggestion
- Anonymizes tweets by replacing real usernames and names with `John Doe` or `Jane Doe`.
```

Comment (sindresorhus) [reply readme.md:45]: This made be realize, it shouldn't replace usernames with `John Doe`. It should rather replace those with `@johndoe` or `@janedoe`. Right?

File: source/features/index.js [hunk 9]
```diff
@@ -195,6 +195,13 @@ export const features = {
 		category: 'timeline',
 		label: 'Adds Markdown-like styling of text wrapped in backticks',
 		fn: require('./inline-code').default
+	},
+	anonymizeTweets: {
+		id: 'feature-anonymize-tweets',
+		category: 'timeline',
+		label: 'Anonymize tweets by replacing real usernames with "John Doe" or "Jane Doe" to focus on tweets\' content',
```
Comment (sindresorhus) [inline source/features/index.js:202]: Same as the readme description comment.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/refined-twitter/pull/173#discussion_r269527458 still needs to be fixed.

**Comment by sindresorhus** [comment]
@iGitScor Bump :)

**Comment by sindresorhus** [comment]
Closing for lack of response. Still happy to land this if you find time to finish it ;)

**Pull request closed without merging**